src/commands/load.rs

```diff
@@ -5,6 +5,13 @@ use crate::types::flag::Flag;
 use crate::types::status::Status;
 use crate::utils;
 
+/// This enum is used to define type of URL (.i.e.. GitHub, GitLab)
+#[derive(Clone)]
+pub(crate) enum URLType {
```

Comment:
Please move that Type to a own file under `src/types`. 
Otherwise the PR looks good and can be merged after that small change :)

Reply:
Made the change here [94b7250](https://github.com/cophilot/templify/pull/14/commits/94b725075d7096a5629f50e3286254d9370df3d2)

@cophilot